packages/select/src/components/query-list/itemRenderer.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import { MouseEventHandler } from "react";
+
+export interface IItemModifiers {
+    /** Whether this is the "active" (focused) item, meaning keyboard interactions will act upon it. */
+    active: boolean;
+
+    /** Whether this item is disabled and should ignore interactions. */
+    disabled: boolean;
+
+    // TODO: not happy with this name
+    /** Whether this item matches the predicate. A typical renderer could hide `false` values. */
+    filtered: boolean;
```

Comment:
Specifically having a "filtered" prop is a little confusing; why is filtering special vs any other thing that users might want to do

Reply:
filtering is one of `QueryList`'s existential features: `itemPredicate` /  `itemListPredicate`.
this API sets us up to support arbitrary user-defined modifiers.